src/FM.LiveSwitch.Mux.Standard/Muxer.cs

```diff
@@ -353,16 +369,15 @@ private async Task<LogEntry[]> GetLogEntries(MuxOptions options)
                 case StrategyType.Hierarchical:
                     {
                         var logFilePath = Path.Combine(options.InputPath, HierarchicalLogFileName);
-                        if (!File.Exists(logFilePath))
+                        if (!File.Exists(logFilePath) || File.Exists($"{logFilePath}.fail"))
```

Comment:
I added logic that should support a hierarchical set up - but in my testing I found the preprocessor doesn't actually support the hierarchical structure so I'm not sure if this section actually gets used.